pkg/testing/open-match/mockpb/mock_frontend_pb.go

```diff
@@ -0,0 +1,633 @@
+// Copyright 2018 Google LLC
```

Comment:
Hi @shin5ok, 
Please chnage the year to 2023 as the Copyright.

Reply:
Thanks for your comment, applied it.